src/dynamo/models/notification.ts

```diff
@@ -26,7 +26,7 @@ const client = createClient({
   ],
 });
 
-module.exports = {
+const notification = {
```

Comment:
Somehow I missed updating the exports to modern style when this was converted to typescript